src/main/java/com/dumptruckman/bukkit/configuration/json/JsonConfiguration.java

```diff
@@ -72,7 +67,7 @@ public void loadFromString(@NotNull final String contents) throws InvalidConfigu
     }
 
     private void convertMapsToSections(@NotNull Map<?, ?> input, @NotNull final ConfigurationSection section) {
-        final Object result = SerializationHelper.deserialize(input);
+        final Object result = SerializationHelper.deserialize(input, options().continueOnSerializationError);
```

Comment:
This should use a getter to be consistent with FileConfigurationOptions.

Reply:
done